src/environmentd/src/bin/environmentd/main.rs

```diff
@@ -101,6 +101,14 @@ pub struct Args {
     ///     legitimate security risk if used in Materialize Cloud.
     #[clap(long, env = "UNSAFE_MODE")]
     unsafe_mode: bool,
+    /// Enable persisted introspection sources.
+    ///
+    /// These sources are temporarily disabled because they put significant
```

Comment:
Reference the Github issue.

Reply:
For the persist compaction issue? Would that be https://github.com/MaterializeInc/materialize/issues/15093?

Reply:
This, or directly the issue this PR is linked to. The latter might provide more of an explanation why we added this flag instead of describing the bigger picture.